src/_pytest/mark/structures.py

```diff
@@ -399,6 +400,9 @@ def store_mark(obj, mark: Mark) -> None:
     assert isinstance(mark, Mark), mark
     # Always reassign name to avoid updating pytestmark in a reference that
     # was only borrowed.
+    if hasattr(obj, "_pytestfixturefunction"):
```

Comment:
A while ago I centralized all of the accesses to this attribute in the `getfixturemarker` function -- can you use it instead?

Reply:
this is tricky because it's a cyclic import, do you want me to move `getfixturemarker` to a new module?

Reply:
The cycle import is real, seems better not to hide it. So I'd import it inside the function instead of at the top level.